app/containers/WalletManager/Wallet.jsx

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import React from 'react'
```

Comment:
Could you also import `{ Component }`

Reply:
Why? It would be an unused import/variable declaration...